tutorials/R/spatial-data/spatialData.Rmd

```diff
@@ -22,331 +23,514 @@ connecting remote sensing with ground-based measurements, the methods in
                               name="",
-                              breaks=unique(sites$Site.Type))
-gg
+                              breaks=unique(neonSites$Site.Type))
+sitesMap
 
 ```
 
 
-### Terrestrial observation plots
+## Map terrestrial observation plots
 
-The locations of observational sampling plots at terrestrial sites (TOS)
-are available in the <a href="http://data.neonscience.org/documents" target="_blank">document library</a> 
-on the Data Portal, in the Spatial Data folder, as static files, 
-in both tabular and shapefile formats. Your download will be a zip file 
-containing tabular files of plot centroids and point locations, and 
-shapefiles of plot centroids, point locations, and polygons.
+The locations of observational sampling plots at terrestrial sites
+are available for download as well from the *NEON Terrestrial Observation System (TOS) sampling locations* file download from the 
+<a href="https://www.neonscience.org/data/spatial-data-maps" target="_blank">Spatial Data and Maps page</a> or from the portal in the 
+<a href="http://data.neonscience.org/documents" target="_blank">document library</a>. 
+The download will be a zip file containing tabular files of plot centroids and 
+point locations, and shapefiles of plot centroids, point locations, and polygons.
 
-The readme file contains descriptions for each of the columns in the 
-tabular files.
+The readme file contains descriptions for each variable in the tabular files. 
 
-```{r TOS-readme, echo=F}
+```{r TOS-readme}
 
-rdme <- read.delim('/Users/clunch/Dropbox/NEON/spatial/All_NEON_TOS_Plots_V5/readme .csv',
+## load TOS plot readme
+rdme <- read.delim('All_NEON_TOS_Plots_V7/readme .csv',
                    sep=',', header=T)
+
+## View the variables
 rdme[,1]
 
 ```
 
-You can use these files to navigate the spatial layout of sampling for 
-TOS: mosquitoes, beetles, plants, birds, etc. In this tutorial, we'll be 
-using the location data provided along with data downloads, as well as 
+You can use these files to plot the spatial layout of sampling for 
+TOS: mosquitoes, beetles, plants, birds, etc. However, for this tutorial, we'll 
+be using the location data provided with data downloads, as well as 
 methods in the `geoNEON` package, to explore TOS spatial data, instead of 
 these files.
 
-## Spatial data in data downloads
+You can, however, modify the code from mapping the domains and field sites to 
+plot the location data if you'd like to do so. 
+
+## Source 2: Spatial data in data downloads
 
-### Observational data
+The second source of data is spatial data in and accompanying data directly 
+downloaded from the NEON data portal. How the data are handled are a bit different 
+for the Observational Sampling data and the Instrumented Systems data, so we'll 
+work through working with both types. 
 
+## Locations for observational data
+
+### Plot level locations
 Both aquatic and terrestrial observational data downloads include spatial 
```

Comment:
There is one minor mention of aquatics sites here but it is pretty much ignored throughout the rest of the tutorial.  Not this iteration but I think we should consider adding in something about location data for aquatic data products.

Reply:
Agreed. I haven't really explored the bathymetry/geomorphology products, but there's probably something cool that could be done there. Could potentially combine with AOP data, too.